packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/simple_auth_configuration.all.bicep

```diff
@@ -0,0 +1,45 @@
+param simpleAuthWebAppName string
```

Comment:
This `simple_auth_configuration.all.bicep` file was removed from dev branch.

Reply:
Removed.